File: CS/CustomSummaryDistinctCountExample/Form1.cs
using DevExpress.XtraEditors;

namespace CustomSummaryDistinctCountExample
{
    public partial class Form1 : XtraForm {
        public Form1() {
            DevExpress.Data.Filtering.CriteriaOperator.RegisterCustomFunction(new DistinctCountFunction());
            InitializeComponent();
            pivotGridControl1.BestFit();
            excelDataSource1.Fill();
       }
    }
}

File: VB/CustomSummaryDistinctCountExample/Form1.vb
Imports DevExpress.XtraEditors

Namespace CustomSummaryDistinctCountExample
	Partial Public Class Form1
		Inherits XtraForm

		Public Sub New()
			DevExpress.Data.Filtering.CriteriaOperator.RegisterCustomFunction(New DistinctCountFunction())
			InitializeComponent()
			pivotGridControl1.BestFit()
			excelDataSource1.Fill()
		End Sub
	End Class
End Namespace
